Updated file

violetadev opened the pull request:
added coach and urls

violetadev committed:
Add files via upload

dv team blogpost

violetadev committed:
Update 2018-07-09-dvteam-activated.md

violetadev committed:
Update 2018-07-09-dvteam-activated.md

violetadev committed:
Update 2018-07-09-dvteam-activated.md

violetadev committed:
Update 2018-07-09-dvteam-activated.md

violetadev committed:
Update 2018-07-09-dvteam-activated.md

Added a small intro to coaches

violetadev committed:
Update 2018-07-09-dvteam-activated.md

added image credit, added header style to Summer and future goals

violetadev committed:
Update 2018-07-09-dvteam-activated.md

Fixed permalink

violetadev committed:
Update 2018-07-09-dvteam-activated.md

Added coach and urls

violetadev committed:
Added image

Added picture of Diana and Violeta

v-thakkar commented on blog/_posts/2018-07-09-dvteam-activated.md:
```diff
@@ -0,0 +1,82 @@
+current: blog
+author: Violeta
+twitter: artnerdnet
+categories: student-posts
```
Hi @artnerdnet ! Thanks for the PR. Please remove 'news' from the category and replace it with 'student-posts'.

v-thakkar commented on blog/_posts/2018-07-09-dvteam-activated.md:
```diff
@@ -0,0 +1,82 @@
+## Starting a new adventure with RgSoC and Living Style Guide from coding newbies to professional front-end developers in 11 months.
+
+![image](https://pbs.twimg.com/media/DiX7VAYX4AEMD2v.jpg)
+
```
Please add image-credits here and for GIF at the end of the file.

v-thakkar commented on blog/_posts/2018-07-09-dvteam-activated.md:
```diff
@@ -0,0 +1,82 @@
+### RgSoC Coaches & Supervisor
+During the Google Scholarship, we met several times in real life and decided to apply for RgSoc to be able to keep practicing our coding skills in summer. As we were new in the dev community, we reached out to our Udacity community managers who introduced us to Amanda Cavallaro. She helped us find our first two current RgSoc-coaches: 
+
+*Coach 1: Tobias (React developer)* 
```
It looks bit weird to have bio for all other coaches except this one. :) Can you please add few lines here as well if you wish to keep mentioning this coach in the blog post?

anasofiapinho commented:
Hello @artnerdnet, can you please make the changes requested by Vaishali to the blog post so that we can publish it? Thank you!

violetadev committed:
Update 2018-07-09-dvteam-activated.md

Fixed the issues and added picture

violetadev committed:
Update 2018-07-09-dvteam-activated.md

Fixed images

violetadev committed:
Update 2018-07-09-dvteam-activated.md

Fixed images

violetadev commented:
@anasofiapinho let me know if this is ok >.< Diana fixed it and I added a pic

v-thakkar commented on blog/_posts/2018-07-09-dvteam-activated.md:
```diff
@@ -0,0 +1,76 @@
+categories: student-posts
+---
+## Starting a new adventure with RgSoC and Living Style Guide from coding newbies to professional front-end developers in 11 months.
+![Starting the summer](http://artnerd.net/images/7452ee6d-ba5c-41b9-8590-dc45c930b256.jpg)
```
@artnerdnet Now that you have added pics in this file, you need to add pic credits as well. Please add them for both pictures in the file.

violetadev replied:
is this needed? it's pictures we took together

v-thakkar replied:
@artnerdnet Yes, I understand it but adding credits is important because bunch of reasons including legal liability. Please check other blog posts for an example on how to add it [basically it's a one line code using image-credits class].

violetadev committed:
Update 2018-07-09-dvteam-activated.md

I added the image credits and also added Bogdan who just joined our team to help close issues.

v-thakkar commented:
Thanks @artnerdnet ! Looks good to me now.

Commit:
Last changes to DV Blog Post

- Changed the date of the blog post
- Added the images to our folder
- Minor formatting changes

anasofiapinho commented:
Merged!
